LocalyticsXamarin/LocalyticsXamarin.iOS/ApiDefinition.cs

```diff
@@ -190,16 +194,16 @@ public interface LLInAppCampaign
 
 		// @property (readonly, getter = isDismissButtonHidden, assign, nonatomic) BOOL dismissButtonHidden;
 		[Export("dismissButtonHidden")]
-		bool GetInAppMessageDismissButtonHidden();
-		//bool DismissButtonHidden { [Bind("isDismissButtonHidden")] get; }
+		//bool GetInAppMessageDismissButtonHidden();
```

Comment:
Why do we need this. The below DismissButtonLocation should suffice right?

Reply:
Nope - DismissButtonLocation tells you left or right - aka what side will the dismiss button be rendered on.  It doesn't tell you if the dismiss button has been hidden.